app/views/rentals/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<h1>Test</h1>
+
+<h3>My requests</h3>
+
+<ul>
+  <% @rentals.each do |rental| %>
+    <% if current_user == rental.user %>
+      <div class="container">
+
+        <%= rental.user.email %>
+
+        <ul>
+          <li>Start Date: <%= rental.start_date %></li>
+          <li>End Date: <%= rental.end_date %></li>
+        </ul>
+  
+        <%= link_to "Cancel this rental", rental_path(rental), method: :delete, data: { confirm: "Are you sure?" } %>
+      </div>
+    <% end %>
+  <% end %>
+</ul>
```

Comment:
if possible, we might be able to iterate over rentals once and filter into the My bookings or Incoming bookings groups during one iteration

Reply:
Not currently working when filtering from the scope.

Currently set in working state with scope filtering in the controller.

Two new instance variables @rentals_user and @rentals_owner have been created. This makes it so the filtering occurs in the controller instead of in the index view